### PR TITLE
readme for develop changes

### PR DESCRIPTION
**Pull request by Kamill90**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,52 +4,55 @@ The first ever Bitcoin Vault wallet.
...
 For iOS:
 GoogleService-Info.plist into directory /ios/
 
 For android:
-google-services.json into directory /android/app/
+google-services.json into directory /config/prod/
+
+- Create .env file:
+
+Create .env file in /config/prod/ directory. It will be copied to root directory while building. Check the .env.example file for a template
```
Comment (KedziaPawel) [inline README.md:27]: there is already .env file in the /config/prod/ so there is no need to create it. 
Can we update SENTRY DSN in it? to 
`SENTRY_DSN=https://dc67fd6d5c2949f2a6853e60eb69d899@o429172.ingest.sentry.io/5375289`

File: README.md [hunk 2]
```diff
@@ -4,52 +4,55 @@ The first ever Bitcoin Vault wallet.
...
-* To run on Android:
+- To run on Android:
 
 ```
-npm start android
+yarn start android:prod
 ```
 
-* To run on iOS:
+if you want to have beta version with different name and id - redo previous steps for config/beta
```
Comment (KedziaPawel) [inline README.md:39]: I think it is not obvious what you gotta do here. I would add something like

To run on Android beta:
Put file google-services.json into directory /config/beta/

> yarn start android:beta

File: README.md (hunk 1 above)
Comment (Kamill90) [reply README.md:27]: So you want to monitor someone else's logs in our sentry?

Comment (KedziaPawel) [reply README.md:27]: you are right, I guess it is the same situation as with google services file. So we should also remove our sentry dsn from env.test

**Review by marek-siemieniuk-morawski — COMMENTED** [review]
Kamil, before merging it please have a look at https://github.com/bitcoinvault/GoldWallet/pull/358

**Pull request closed without merging**